config/webpack/webpack.config.prod.js

```diff
@@ -0,0 +1,23 @@
+const webpack = require('webpack');
+const merge = require('webpack-merge');
+const commonWebpackConfig = require('./webpack.config.common');
+const helpers = require('../helpers');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const { prodLoaders } = require('./loaders');
+
+module.exports = merge({
+  // Prepend new config sections for arrays
+  customizeArray: (commonConfig, newConfig) => [...newConfig, ...commonConfig],
+})(commonWebpackConfig, {
+  devtool: 'source-map',
```

Comment:
change to `cheap-module-source-map` I remember that is better to Production [check](https://github.com/Lemoncode/webpack-2.x-by-sample/blob/master/03%20Misc/06%20Production%20Configuration/prod.webpack.config.js)

Here we have some info